recipe/meta.yaml

```
@@ -32,8 +34,13 @@ test:
 about:
   home: https://github.com/rsalmei/about-time
   summary: Easily measure timing and throughput of code blocks, with beautiful human friendly representations.
+  description: |
+    A cool helper for tracking time and throughput of code blocks, with beautiful human friendly renditions.
   license: MIT
+  license_family: MIT
   license_file: LICENSE
+  doc_url: https://github.com/rsalmei/about-time
+  dev_url: https://github.com/rsalmei/about-time/blob/main/README.md
```

Let's switch these (:
```suggestion
  dev_url: https://github.com/rsalmei/about-time
  doc_url: https://github.com/rsalmei/about-time/blob/main/README.md
```

---

woops! Nice catch! Thank you